notifications/notifications.yml

```diff
@@ -91,6 +91,10 @@ Parameters:
     Type: String
     Description: >
       The name of the main CD pipeline in CodePipeline
+  FatalNotificationEmailAddress:
```

Comment:
So what will this actually be set to?

Reply:
It's deployed if you want to check. Not going to post the address because it's a new one and may as well stay private